_posts/2023-02-14-whatsnew-reporting-cli.markdown

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title:  "What’s new: OpenSearch Reporting CLI"
+authors:
+- maharup
+- jdbright
+- jadhanir
+- lijshu
+- alicejw
+date: 2023-02-14 00:00:00 -0700
+categories:
+ - technical-post
+meta_keywords: "Reporting CLI, OpenSearch Reporting plugin, automate reporting in OpenSearch"
+ meta_description: "Learn how the Reporting CLI makes it convenient to generate external reports without logging in to OpenSearch Dashboards."
+---
+
+The OpenSearch Project is happy to announce the release of the Reporting CLI, making it convenient to generate reports externally without logging in to OpenSearch Dashboards. This gives you the flexibility to programmatically generate reports and connect them to your preferred messaging system. You can now schedule and transport reports efficiently through email.
+
+In this post we’ll show you a few examples of how to leverage the capabilities of the Reporting CLI, such as:  
+
+* Generating a PDF from an eCommerce dashboard that shows data visualizations and scheduling the report to run at a specific time using cron.
+* Incorporating report generation into your workflow with email or notifications.
+* Generating a CSV report from tabular data in the Dashboards **Discover** application.
+
+### What is the Reporting CLI?
+
+The Reporting CLI is a package you can install separately from the Reporting plugin that allows you to specify an OpenSearch Dashboards **Discover** URL and generate a report in PDF or PNG format. You can download reports you generate locally or send reports programmatically to downstream messaging systems. You can also leverage the capabilities of the Reporting CLI to email CSV reports, but that requires the Reporting plugin.
+
+### What can you do with the Reporting CLI?
+
+The Reporting CLI can save you time by automating report generation and distribution across messaging systems and provides access to the report data through several authentication options.
+
+If you want more flexibility in how you can distribute reports throughout your organization through multiple messaging systems, such as Slack or SES email systems, you can conveniently schedule and deliver those Dashboard **Discover** reports with the Reporting CLI.
+
+If you want to use an AWS service event to trigger a report, you can configure a trigger using [AWS Lambda](https://aws.amazon.com/lambda/).
+
+### Example: Generating a PDF from a Dashboard
+
+You can generate an eCommerce revenue Dashboard PDF report to show all of the data visualizations, and schedule the report to run at a specific day and time with the cron utility.
+
+![Image: An example Dashboard visualization PDF]({{site.baseurl}}/assets/media/blog-images/2023-02-14-whatsnew-reporting-cli/cli-pdf-report.png){:.img-fluid }
+
+### Example: Using advanced options to add the Reporting CLI to your workflow*
```

Comment:
Is the asterisk a typo?

Reply:
sorry. my manual changes did not get committed. Thanks for your patience.